docs-src/references/dynamic-configuration.md

```diff
@@ -0,0 +1,173 @@
+Th, the default values will apply. Also, setting custom retry polici
+
+| Dynamic configuration key            | Type                          | Description                                                                                                                   | Default value                                                                      |
+| ------------------------------------ | ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| `history.defaultActivityRetryPolicy` | Map (key-value pair elements) | Out-of-the-box server configuration for an activity retry policy when it is not explicitly set for the Activity in your code. | [Default values for retry Policy](/retry-policies#default-values-for-retry-policy) |
+| `history.defaultWorkflowRetryPolicy` | Map (key-value pair elements) | Out-of-box Retry Policy for unset fields where the user has set an explicit `RetryPolicy`, but not specified all the fields   | [Default values for retry Policy](/retry-policies#default-values-for-retry-policy) |
+
+### Size limit settings
+
+The Persistence database in the Cluster has default size limits set for optimal performance. The dynamic configuration keys relating to some of these are listed below.
+
+The default values on these keys have been set based on extensive testing. While these values can be changed, ensure that you are provisioning enough database resources to handle the changed values.
+
+| Dynamic configuration key               | Type | Description                                                                                                                                                                                                                                | Default value             |
+| --------------------------------------- | ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------- |
+| `limit.maxIDLength`                     | Int  | Length limit for various IDs, including: `Namespace`, `TaskQueue`, `WorkflowID`, `ActivityID`, `TimerID`, `WorkflowType`, `ActivityType`, `SignalName`, `MarkerName`, `ErrorReason`/`FailureReason`/`CancelCause`, `Identity`, `RequestID` | 1000                      |
+| `system.transactionSizeLimit`           | Int  | Largest allowed transaction size to the Persistence database.                                                                                                                                                                              | 4 MB (`4 * 1024 * 1024`)  |
```

Comment:
This config is confusing, even for server team... 
It controls the maximum size of each history event batch. Usually people should not touch it.

Reply:
Should I remove it? Or maybe I can add some more detail explaining when to change this value (if at all) and what to consider before doing so?

Reply:
I would remove it, the value should never be changed unless something really bad happens. We have this [fix](https://github.com/temporalio/temporal/pull/4296) released, which addresses the only case we have in cloud where we need to tune this value for mitigation.

Reply:
removed